src/web/demo.rkt

```diff
@@ -239,7 +239,15 @@
 
             (eprintf "Job ~a complete\n" hash)
             (hash-remove! *jobs* hash)
-            (semaphore-post sema)])])
+            (semaphore-post sema)])]
+            [(list 'sample _hash formula sema _seed)
```

Comment:
Indentation please fix your editor

Reply:
Also why `_seed`? Since it is actually used?

Reply:
Fixed indentation I think. 

Was trying to be explicit about which seed is being used. Can change `_hash` and `_seed` to remove the underscore.

Reply:
Yeah remove the underscore. That's usually used to mean "this field is ignored, but to help you keep track, this is the seed"